### PR TITLE
Update scrape TG to latest page version

### PR DESCRIPTION
**Pull request by dominikgehl**

### CODE REVIEW

**Comment by baryluk** [comment]
Please update scraper_matrix.py

Also, would be nice to see a better description and title of the PR.

**Comment by dominikgehl** [comment]
@baryluk PR title and scrape_matrix.py are updated.

**Comment by baryluk** [comment]
@dominikgehl Great. Do you also see the comment I did on the `scrape_tg.py` file? About the regexp.

**Comment by metaodi** [comment]
@baryluk I don't see your comment  neither. Make sure you submit your review after making the comment. Otherwise it's only visible for you.

**Comment by baryluk** [comment]
@metaodi Ok. I see, when you "start a review", you can do comments on code, in many places, and then send them as a one "review". Make sense actually. It was not obvious from the buttons tho.

File: scrapers/scrape_tg.py [hunk 1]
```diff
@@ -27,4 +27,9 @@
 
 print('Date and time:', sc.find(r'Stand\s*([^<]+)<', d))
 print('Confirmed cases:', sc.find(r'(?:Anzahl)?\s*bestätigter\s*Fälle:?\s*([0-9]+)\b', d))
-print('Deaths:', sc.find(r'\b([0-9]+)\s*verstorb', d))
+deaths = sc.find(r'\b([0-9]+)\s*verstorb', d)
+if deaths is None:
+    deaths = sc.find(r'Verstorben: ([0-9]+)', d)
```
[the remark this answers is not in the document]
Comment (dominikgehl) [reply scrapers/scrape_tg.py:32]: Ok. I'll update it

**Comment by baryluk** [comment]
LGTM

**Pull request merged**